lib/vaahextendflutter/widgets/atoms/input_text.dart

```diff
@@ -123,14 +125,29 @@ class InputText extends StatelessWidget {
     );
   }
 
-  OutlineInputBorder border(Color color) {
-    return OutlineInputBorder(
-      borderRadius: BorderRadius.circular(borderRadius),
-      borderSide: BorderSide(
-        width: 1,
-        color: color,
-      ),
-    );
+  InputBorder getInputBorder(InputBorderType inputBorderType, Color color) {
+    switch (inputBorderType) {
+      case InputBorderType.none:
+        return InputBorder.none;
+
+      case InputBorderType.underline:
+        return UnderlineInputBorder(
+          borderRadius: BorderRadius.circular(borderRadius),
+          borderSide: BorderSide(
+            width: 1,
+            color: color,
+          ),
+        );
+
+      case InputBorderType.outline:
+        return OutlineInputBorder(
+          borderRadius: BorderRadius.circular(borderRadius),
+          borderSide: BorderSide(
+            width: 1,
```

Comment:
everything looks good, do we also want to control width in the future like we did for  below?
```dart
this.inputBorder = InputBorderType.outline,
```

Reply:
May be....but most of the times user keeps the default width, alters it If the requirement arises because of design.